AppBuilder/platform/dataFields/ABFieldConnect.js

```diff
@@ -335,6 +335,8 @@ module.exports = class ABFieldConnect extends ABFieldConnectCore {
 
          Promise.resolve()
             .then(async () => {
+               // disabling local storage of options
```

Comment:
Can we add why we're disabling it, just so we can understand when we come back to this in the future?

Reply:
👍